pkg/watchdog/watchdog-report.sh

```diff
@@ -9,7 +9,7 @@
 
 # First log to /persist in case zboot/kernel is hung on disk
 
-DATE=$(date -Ins)
+DATE=$(date -Is)
```

Comment:
Why remove the sub-second granularity?

Reply:
This simply a side effect of busybox date implementation only supporting seconds. That said, I think we should simply remove this timestamping part from the services output altogether. It is annoying when you see it on the console (when running by hand) and it is getting timestamped with high precision in memlogd anyway.

WDYT @nordmark ?

Reply:
Which date are we using from device-steps.sh? Ah it's coreutils.
For the watchdog-related stuff the sub-seconds don't matter. I haven't looked what granulatiry we get from memlogd/rsyslogd so it might not be time to remove the logrus nanoseconds yet. But this is fine.